src/main/webapp/gameFunctions.js

```diff
@@ -39,10 +47,39 @@ function getUserQuizStatus() {
     fetch("/user-quiz-status-servlet").then(response => response.json()).then((user_has_taken_quiz) => {
         if(!user_has_taken_quiz){
             document.getElementById("QuizButton").style.display = "block";
+        } else{
+             document.getElementById("QuizButton").style.display = "none";
         }
     });
 }
-
+ 
+function populateQuizPage(){
+    var name_container = document.getElementById("userName");
+    var score_container = document.getElementById("score");
+    var image_holder = document.getElementById("image-holder");
+    var gameId_container = document.getElementById("game-board-id");
+ 
+    fetch("/populate-quiz-page").then(response => response.json()).then(data => {
+        name_container.innerHTML = data.propertyMap.username;
+        score_container.innerHTML = data.propertyMap.score;
+        gameId_container.innerHTML = ("Game Id: " + data.propertyMap.gameId);
+        if(data.propertyMap.blobKey != null){
+            makeImage(image_holder, data.propertyMap.blobKey);
+        }
+    }
+    )
```

Comment:
This is super nit picky but we can combine lines 69 and 70 into one line (like you did on line 79)